Explorer/Assets/DCL/PluginSystem/Global/DynamicSettings.cs

```diff
@@ -21,5 +22,6 @@ public class DynamicSettings : IDCLPluginSettings
         [field: SerializeField] public AssetReferenceT<AudioMixer> GeneralAudioMixer { get; private set; }
         [field: SerializeField] public AssetReferenceT<MultiplayerDebugSettings> MultiplayerDebugSettings { get; private set; }
 
+        [field: SerializeField] public AssetReferenceGameObject AppVerRedirectionScreenPrefab { get; private set; }

```

Comment:
why is this a dynamic thing? can't we have a plugin initialising the resources with its own configuration?

Reply:
actually, it would be nice to put this logic a bit earlier then after plugin initialization (as early as we can do). I just didn't have time. That is why keeping it in DynamicSettings make sense